tests/analytical_comparisons/test_analytical.py

```diff
@@ -56,27 +71,29 @@ def test_analytical(name, expected, config):
     b = Path(__file__).parent.resolve()
 
     dats = [
-        pd.read_csv(b / "errors-{}-levels{}-{}.dat".format(
-            name.replace("/", "_"),
-            level,
-            idfn(config)
-        ), sep=" ", header=None)
+        pd.read_csv(
+            b
+            / "errors-{}-levels{}-{}.dat".format(
+                name.replace("/", "_"), level, idfn(config)
+            ),
+            sep=" ",
+            header=None,
+        )
         for level in levels
     ]
 
-    # Number of variables
-    variables = 3
-    # free surface deals with a eta, else with normal stress
-    if name.split("_")[-1] == "freesurface":
-        columns = ["l2error_u", "l2error_p", "l2error_eta", "l2anal_u", "l2anal_p", "l2anal_eta"]
+    cols_anal = ["l2anal_u", "l2anal_p"]
+    cols_err = ["l2error_u", "l2error_p"]
+    if name.split("/")[-1] == "free_surface":
```

Comment:
This doesn't seem correct? I think it should be as it was:

```python3
    if name.split("_")[-1] == "freesurface":
```

Reply:
My bad, it looks like I missed that change from gh-131. Thank you for spotting it.